File: Backend/src/FunnyCode/Models/DTO/Responses/Event.cs
namespace FunnyCode.Domain.Core.Entities;

public class Event
{
    public Guid Id { get; set; }
    public List<Subdivision> Subdivisions { get; set; } = new();
    public List<Division> Divisions { get; set; } = new();
    public DateTime Date { get; set; }
    public string Description { get; set; }
    public string Location { get; set; }
    public string Name { get; set; }    

}

File: Mobile/Android/data/src/main/java/com/example/funnyapp/data/model/response/Event.kt
package com.example.funnyapp.data.model.response

data class Event(
    val Id: String,
    val Name: String,
    val Description: String,
    val Date: String,
    val Location: String,
    val Subdivisions: List<Subdivisions>,
    val Divisions: List<Divisions>
)
